components/compass-runtime-agent/internal/compass/director/converter_test.go

```diff
@@ -339,85 +335,36 @@ func fixInternalAPIBundleEmptySpecs(suffix string) kymaModel.APIBundle {
 		Description:                    stringPtr(baseBundleDesc + suffix),
 		InstanceAuthRequestInputSchema: stringPtr(baseBundleInputSchema + suffix),
 		APIDefinitions: []kymaModel.APIDefinition{
-			fixInternalAPIDefinition("1", nil, nil),
-			fixInternalAPIDefinition("2", nil, nil),
-			fixInternalAPIDefinition("3", nil, nil),
-			fixInternalAPIDefinition("4", nil, nil),
+			fixInternalAPIDefinition("1", nil),
+			fixInternalAPIDefinition("2", nil),
+			fixInternalAPIDefinition("3", nil),
+			fixInternalAPIDefinition("4", nil),
 		},
 		EventDefinitions: []kymaModel.EventAPIDefinition{
-			fixInternalEventAPIDefinition("1", nil),
-			fixInternalEventAPIDefinition("2", nil),
-		},
-		Documents: []kymaModel.Document{
-			fixInternalDocument("1", nil),
-			fixInternalDocument("2", nil),
-			fixInternalDocument("3", nil),
+			fixInternalEventAPIDefinition("1"),
+			fixInternalEventAPIDefinition("2"),
 		},
 	}
 }
 
-func fixInternalAPIDefinition(suffix string, credentials *kymaModel.Credentials, spec *kymaModel.APISpec) kymaModel.APIDefinition {
+func fixInternalAPIDefinition(suffix string, credentials *kymaModel.Credentials) kymaModel.APIDefinition {
 	return kymaModel.APIDefinition{
 		ID:          baseAPIId + suffix,
 		Name:        baseAPIName + suffix,
 		Description: baseAPIDesc + suffix,
 		TargetUrl:   baseAPIURL + suffix,
-		APISpec:     spec,
 		Credentials: credentials,
 	}
 }
 
-func fixInternalEventAPIDefinition(suffix string, spec *kymaModel.EventAPISpec) kymaModel.EventAPIDefinition {
+func fixInternalEventAPIDefinition(suffix string) kymaModel.EventAPIDefinition {
 	return kymaModel.EventAPIDefinition{
-		ID:           baseAPIId + suffix,
-		Name:         baseAPIName + suffix,
-		Description:  baseAPIDesc + suffix,
-		EventAPISpec: spec,
-	}
-}
-
-func fixInternalDocument(suffix string, data []byte) kymaModel.Document {
-	kind := baseDocKind + suffix
-
-	return kymaModel.Document{
 		ID:          baseAPIId + suffix,
+		Name:        baseAPIName + suffix,
 		Description: baseAPIDesc + suffix,
-		Title:       baseDocTitle + suffix,
-		DisplayName: baseDocDisplayName + suffix,
-		Format:      kymaModel.DocumentFormatMarkdown,
-		Kind:        &kind,
-		Data:        data,
 	}
 }
 
```

Comment:
Please remove Documents from function below member since we are not processing it anymore